plugins/modules/iam_user_info.py

```diff
@@ -103,15 +103,26 @@
             type: dict
             returned: if user exists
             sample: '{"Env": "Prod"}'
+        console_access:
+            description: If user has access to log in from AWS default console.
+            returned: always
+            type: bool
+            sample: "true"
 """
 
 from ansible_collections.amazon.aws.plugins.module_utils.iam import AnsibleIAMError
 from ansible_collections.amazon.aws.plugins.module_utils.iam import get_iam_group
 from ansible_collections.amazon.aws.plugins.module_utils.iam import get_iam_user
 from ansible_collections.amazon.aws.plugins.module_utils.iam import list_iam_users
 from ansible_collections.amazon.aws.plugins.module_utils.iam import normalize_iam_user
-from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule, is_boto3_error_code
```

Comment:
```suggestion
from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
```

We use a single import per-line formatting standard, this helps reduce merge conflicts when multiple changes are in-flight.